packages/web-components/fast-components/src/color-2/utilities/is-dark-mode.ts

```diff
@@ -0,0 +1,18 @@
+import { Swatch } from "../swatch";
+
+/*
+ * A color is in dark mode if there is more contrast between #000000 and a background
+ * color than #FFFFFF and a background color. That threshold can be expressed as a relative luminance
+ * using the contrast formula as (1 + 0.5) / (bg + 0.05) === (bg + 0.05) / (0 + 0.05),
+ * which reduces to the following, where bg is the relative luminance of the background color
+ */
+const target = (-0.1 + Math.sqrt(0.21)) / 2;
+
+/**
+ * Determines if a color should be considered Dark Mode
+ * @param color - The color to check to mode of
+ * @returns boolean
+ */
+export function isDarkMode(color: Swatch): boolean {
```

Comment:
I touched on this in `Palette` above, but I feel like the name of this function doesn't encourage correct usage. It's a concise name though. More accurately it's something like `isCloserToBlack`. Horrible name, but I'm open to the idea of removing this from the notion of light or dark mode UI.